airbyte-cdk/python/setup.py

```diff
@@ -13,6 +13,8 @@
 # The text of the README file
 README = (HERE / "README.md").read_text()
 
+pyarrow_dependency = "pyarrow==12.0.1"
```

Comment:
For my knowledge since I had this question when I was debugging this earlier, are we explicitly pinning to 12.0.1 for a reason or does 12.0.0 work just the same?

Reply:
There's a mix of ~= and == and I'm not sure what's intentional

Reply:
I'd expect 12.0.0 to also work. Will revisit in a follow up PR